app/api/route/schemas/user_schemas.py

```diff
@@ -4,7 +4,8 @@
 from apiflask import fields
 from marshmallow import fields as marshmallow_fields
 
-from api.db.models.user_models import Role, RoleType
+from api.db.models.user_models import Role, RoleType, User
+from api.services.models.user import UserPatchParams
```

Comment:
good call putting `UserPatchParams` in the services package. i think that makes more sense than in the `route` folder like i had it in [my PR](https://github.com/navapbc/template-application-flask/pull/57/files) since the services package shouldn't depend on anything specific to the web/api context.

on another note, i suggest we do
```
from api.services.models import user
```
and then use `user.UserPatchParams` so we don't accidentally re-export `UserPatchParams` from this module (see my other comment about that)